packages/network-controller/src/index.ts

```diff
@@ -30,6 +30,7 @@ export type {
   NetworkControllerSetActiveNetworkAction,
   NetworkControllerAddNetworkAction,
   NetworkControllerRemoveNetworkAction,
+  NetworkControllerUpdateNetworkAction,
   NetworkControllerDangerouslySetNetworkConfigurationAction,
```

Comment:
second commit removes this.

This is a temp branch, so will split this into a separate PR.